ister.py

```diff
@@ -400,10 +400,13 @@ def copy_os(args, template, target_dir):
         run_command("mount --bind {0}/var/tmp /var/lib/swupd"
                     .format(target_dir))
     swupd_env = os.environ
-    if template.get("Proxy"):
-        swupd_env["http_proxy"] = template["Proxy"]
-        swupd_env["https_proxy"] = template["Proxy"]
-        LOG.debug("proxy: {}".format(template["Proxy"]))
+    if template.get("HTTPSProxy"):
+        swupd_env["https_proxy"] = template["HTTPSProxy"]
+        LOG.debug("https_proxy: {}".format(template["HTTPSProxy"]))
+
+    if template.get("HTTPProxy"):
+        swupd_env["http_proxy"] = template["HTTPProxy"]
+        LOG.debug("http_proxy: {}".format(template["HTTPProxy"]))
```

Comment:
Hrm just realized that the template change should be added to the validate_template function (and tests added for that too).